src/components/elements/storyTranslationAndCitation.tsx

```diff
@@ -4,9 +4,18 @@ export const StoryTranslationAndCitation = (props: any) => {
     const story = props.story;
 
     // Function to write english translation author, if any
-    const ConstructEnglishTranslationAuthor = (english_translation_author: string, english_translation_manuscript_name: string, english_translation_manuscript_folio: string) => {
+    const ConstructEnglishTranslationAuthor = (english_translation_author: string, english_translation_manuscript_name: string, english_translation_manuscript_folio: string, translations: any) => {
+
+        let translation_date = null
+        for (let tran of translations){
+            if (tran.language_translated_to.toLowerCase() == "english"){
```

Comment:
Converted this to be lowercase so that we can be less likely we don't miss this because of capitalization. Pull request looks good, I think you should test it with prod data with Issac like he suggested.